src/reactivity/readonly.ts

```diff
@@ -44,6 +44,11 @@ type UnwrapNestedRefs<T> = T extends Ref ? T : UnwrapRef<T>
 export function readonly<T extends object>(
   target: T
 ): DeepReadonly<UnwrapNestedRefs<T>> {
+  if (!isObject(target)) {
+    if (__DEV__) {
```

Comment:
This API is provided for type level only, it's better to judge `__DEV__` first

Reply:
sure